sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -396,7 +620,7 @@ internal List<ArraySegment<byte>> ConvertLockTokensToDeliveryTags(IEnumerable<Gu
         ///
         /// <param name="sessionId"></param>
         /// <param name="timeout"></param>
-        public override async Task<DateTime> RenewSessionLockAsync(
+        internal async Task<DateTime> RenewSessionLockInternal(
             string sessionId,
```

Comment:
Can we remove the `sessionId` param and use the class property?

Reply:
You bet